javascript/javascript3/week3/promise_main.js

```diff
@@ -0,0 +1,33 @@
+//Getting into promises
+
+let promiseAll = Promise.all([
```

Comment:
Nice, although if you are not planning to reassign the values later, favour using const. 💃